src/main/java/com/example/ibeproject/service/GraphQLService.java

```diff
@@ -0,0 +1,34 @@
+package com.example.ibeproject.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.*;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestClientException;
+import org.springframework.web.client.RestTemplate;
+
+
+@Service
+public class GraphQLService {
+    private static final String ROOMS_DATA = "{ listRooms {room_id room_number } }";
+    private static final String APIKEY = "da2-fakeApiId123456";
```

Comment:
Put it in application.properties
Don't hardcode anything here

Reply:
Done